_template_python/delphi_NAME/run.py

```diff
@@ -25,11 +32,46 @@ def run_module(params):
     --------
     params:  Dict[str, Any]
         Nested dictionary of parameters.
-
-    Returns
-    -------
-    prints the updated signal names
     """
-    wip_signal = params["indicator"]["wip_signal"]
-    signal_names = add_prefix(SIGNALS, wip_signal, prefix="wip_")
-    print(signal_names)
+    start_time = time.time()
+    logger = get_structured_logger(
+        __name__, filename=params["common"].get("log_filename"),
+        log_exceptions=params["common"].get("log_exceptions", True))
+    mapper = GeoMapper()
+    stats = []
```

Comment:
low pri, but might want to consider renaming this to `run_stats` or something else which makes it clear that it's related to the job and not some statistics for the signal